### ISSUE TITLE
Resolve nil panic

### ISSUE THREAD

**Issue by cznic** [cznic/cc#95]
bypassed here: https://github.com/cznic/cc/commit/563948c33bbc4e7373939ff23637f66cc3131cd0

Updates cznic/virtual#1

**Comment by steffengy** [comment]
Since I didn't see this issue at first, the same comment here:

m.TypeQualifier == nil => true

**Comment by cznic** [comment]
Yeah. Maybe next weekend. You're more than welcome to dig into it before me.
